conda/recipes/mantid/meta.yaml

```diff
@@ -59,6 +59,7 @@ requirements:
     - {{ pin_compatible("numpy", upper_bound="1.25") }}
     - {{ pin_compatible("occt", max_pin="x.x.x") }}
     - pycifrw {{ pycifrw }}
+    - pystog {{ pystog }}
```

Comment:
Since this adds to our conda recipe, can we test that the packaging still works with this change?